docs/content/docs/connectors/table/datagen.md

```diff
@@ -270,6 +270,13 @@ Connector Options
       <td>Long</td>
       <td>The total number of rows to emit. By default, the table is unbounded.</td>
     </tr>
+    <tr>
+      <td><h5>scan.parallelism</h5></td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">(none)</td>
+      <td>Long</td>
```

Comment:
Actually it's defined as `int` in the code.

Reply:
Thanks for catching this. I'll fix that

Reply:
Fixed in the new commit